tests/e2e/vsan_stretched_cluster.go

```diff
@@ -3290,4 +3302,674 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		err = waitForAllNodes2BeReady(ctx, client)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
+
+	/*
+			Site failover during full sync
+			Steps:
+			1.  Configure a vanilla multi-master K8s cluster with inter and intra site replication
+			2.	Create 6 PVCs with reclaim policy Delete and 8 with reclaim policy Retain
+				and wait for them to be bound
+			3.	Delete four PVCs with reclaim policy Retain
+			4.	Delete two PVs reclaim policy Retain related to PVC used in step 3
+			5.	Create two pods using PVCs with reclaim policy Delete
+			6.	Bring vsan-health service down
+			7.	Create two pods with two PVCs each
+			8.	Create two static PVs with disk left after step 4
+			9.	Create two PVCs to bind to PVs with reclaim policy Retain
+			10. Delete four PVCs with reclaim policy Retain different from the ones used in step 3 and 9
+			11. Delete two PVs reclaim policy Retain related to PVC used in step 10
+			12. Add labels to all PVs, PVCs
+			13. Bring vsan-health service up when full sync is triggered
+			14. Bring down primary site
+			15. Verify that the VMs on the primary site are started up on the other esx servers
+		        in the secondary site
+			16. Wait for full sync
+			17. Verify CNS entries
+			18. Delete all pods, PVCs and PVs
+			19. Bring primary site up and wait for testbed to be back to normal
+
+	*/
+	ginkgo.It("[primary-centric] Site failover during full sync", func() {
```

Comment:
this test was there for 'distributed' case as well right?

Reply:
It was for single replica 'distributed' test, hence have only put 'primary-centric' here.